WcaOnRails/config/locales/de.yml

```diff
@@ -1010,6 +1032,29 @@ de:
           Eventkombinationen verbieten willst. Bitte fülle dies in Englisch aus!
         #original_hash: 1067798
         main_event_id: Der Gewinner dieses Events ist der Gewinner der Competition.
+        #original_hash: da39a3e
+        allow_registration_without_qualification: ''
+        championships:
+          #original_hash: da39a3e
+          championship_type: ''
+        competition_series:
+          #original_hash: da39a3e
+          wcif_id: ''
+          #original_hash: 77811b5
+          name: >-
+            Der volle Name der Series Competition, inklusive des Jahres am Ende
```

Comment:
In this instance you opted for the anglicism, but further down you translate it as "Wettbewerbsserie". Please pick either one and make it consistent :)

Reply:
Will do 😄